### ISSUE TITLE
JS Basic Grading

### ISSUE THREAD

**Issue by wesharper** [webfun-sept2017/mbrady_webfun-sept2017#1]
There's no content!
https://github.com/webfun-sept2017/mbrady_webfun-sept2017/blob/1709e20f8ed1a7f25566bc6686ca40a4ec49e4a7/WebFundamentals/HTML/javascript_basic_blocks.html#L15-L25

**Comment by mattgbrady** [comment]
Yes, sorry. For some reason just did the layout. Can I go back and add?

**Comment by nguyenhmp** [comment]
its okay matt, you have already started on the other ones, no point in doing a more basic assignment!

**Comment by wesharper** [comment]
Next time just use some filler text because some of the more challenging styling in this assignment had to do with the text.
